src/components/RocketContainer.js

```diff
@@ -1,18 +1,65 @@
 /* eslint-disable react/prop-types */
-/* eslint-disable max-len */
```

Comment:
Kindly consider not disabling the linters for they help to write a clean and standardized code.